content/codewars-python-test-framework-v2.md

```diff
@@ -0,0 +1,298 @@
+# Python test framework for Codewars (v2)
+(upgrade by [@Bubbler-4](https://github.com/Bubbler-4), Feb 2019)
+
+<br>
+<br>
+
+You may find here the detailed information about the [test framework](https://github.com/codewars/python-test-framework/blob/master/codewars_test/test_framework.py) designed for CodeWars. The present version is extending the first one, so old katas are still compatible, but the use of the v2 is required for newly created katas or translations.
+
+_Note: the test module is aliased with `test`, so you can call every tool provided with it using either `Test.method` or `test.method`._
+
+<br>
+<br>
+
+---
+
+## Overview
+
+The test framework allowes to write named groups of tests holding other named groups of tests, containing themselves assertions.
+
+* Assertions (using the methods detailed below) can actually be used at any level in the groups, or even outise of them (but you're advised to avoid that).
+* Note that the code executed inside on group isn't stopped at the first failed assertion (unlees you use `allow_raise`. See below), so the full test suite is executed each time.
+
+The basic setup for the tests follow this example:
+
+```python
+@Test.describe('Fixed Tests')
```

Comment:
Can you change to `@test`?

We'll continue to support `Test` for now, but in the future, I'd like to let users just import the test framework normally like `import codewars_test as test` (removing the runner magic). So I want the documentation to constantly use `test`.

Reply:
argh... I just did the complete opposite x)

Ok.

Reply:
done (the markdowns too)